### PR TITLE
fix(minio): fix polling working half the time because of continuation…

### PR DESCRIPTION
**Pull request by jbordeau**
There is an error after the first polling, because the continuousToken keep the last file polled. So the next polling do nothing. We have to wait the next to works again and that continue one in two.
I can't create jira ticket ?

- [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [ ] If you're unsure, you can format the pull request title like `[CAMEL-XXX] Fixes bug in camel-file component`, where you replace `CAMEL-XXX` with the appropriate JIRA issue.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Run `mvn clean install -Psourcecheck` in your module with source check enabled to make sure basic checks pass and there are no checkstyle violations. A more thorough check will be performed on your pull request automatically.
Below are the contribution guidelines:
https://github.com/apache/camel/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: components/camel-minio/src/main/java/org/apache/camel/component/minio/MinioConsumer.java [hunk 1]
```diff
@@ -68,6 +69,7 @@ public class MinioConsumer extends ScheduledBatchPollingConsumer {
 
     public MinioConsumer(MinioEndpoint endpoint, Processor processor) {
         super(endpoint, processor);
+        LOG.info("Minio consumer custom: v1");
```
Comment (davsclaus) [inline components/camel-minio/src/main/java/org/apache/camel/component/minio/MinioConsumer.java:72]: Can you remove this

File: components/camel-minio/src/test/resources/log4j2.properties [hunk 2]
```diff
@@ -24,4 +24,5 @@ appender.out.name=out
 appender.out.layout.type=PatternLayout
 appender.out.layout.pattern=%d [%-15.15t] %-5p %-30.30c{1} - %m%n
 rootLogger.level=INFO
-rootLogger.appenderRef.file.ref=file
+rootLogger.appenderRef.out.ref=out
```
Comment (davsclaus) [inline components/camel-minio/src/test/resources/log4j2.properties:27]: Can you revert this

Comment (davsclaus) [inline components/camel-minio/src/test/resources/log4j2.properties:27]: Can you revert this

**Review by davsclaus — APPROVED** [review]

File: components/camel-minio/src/test/java/org/apache/camel/component/minio/MinioTestUtils.java [hunk 3]
```diff
@@ -37,17 +37,4 @@ public static Properties loadMinioPropertiesFile() throws IOException {
 
         return properties;
     }
-
-    static Properties loadMinioAccessFromJvmEnv() throws Exception {
```
Comment (oscerd) [inline components/camel-minio/src/test/java/org/apache/camel/component/minio/MinioTestUtils.java:41]: Is there a specific reason to remove this?

**Review by oscerd — CHANGES_REQUESTED** [review]
Minimal finding.

File: components/camel-minio/src/test/java/org/apache/camel/component/minio/MinioTestUtils.java (hunk 3 above)
Comment (jbordeau) [reply components/camel-minio/src/test/java/org/apache/camel/component/minio/MinioTestUtils.java:41]: it is not used. Do you want keep it ?

File: components/camel-minio/src/main/java/org/apache/camel/component/minio/MinioConsumer.java (hunk 1 above)
Comment (jbordeau) [reply components/camel-minio/src/main/java/org/apache/camel/component/minio/MinioConsumer.java]: done, it is an error.

File: components/camel-minio/src/test/resources/log4j2.properties (hunk 2 above)
Comment (jbordeau) [reply components/camel-minio/src/test/resources/log4j2.properties]: done

Comment (jbordeau) [reply components/camel-minio/src/test/resources/log4j2.properties]: done

**Review by davsclaus — APPROVED** [review]

**Pull request merged**